tmt/tests/system-reinstall/test-system-reinstall-bootc.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+set -exuo pipefail
+
+# Check environment
+printenv
+
+# This script only runs on Packit and gating environment
+if [ "$TMT_REBOOT_COUNT" -eq 0 ]; then
+    pwd
+    # change to hack directory
+    cd ../../hack
+
+    ARCH=$(uname -m)
+    # Get OS info
+    source /etc/os-release
+    case "$ID" in
+        "centos")
+            BASE="quay.io/centos-bootc/centos-bootc:stream${VERSION_ID}"
```

Comment:
We keep adding new copies of this...I'd like to consolidate that. here's an random idea. How about we add a field `BOOTC_IMAGE` into `os-release`?

But in the intermediate term, I think we could probably have a `lib.sh` or something that's shared with `build.sh` too that we use here?

A different idea: Move all the tmt stuff into tests/ and have this "image mapping helper" live in tests?

Reply:
@cgwalters , What about this https://github.com/bootc-dev/bootc/pull/1642/files#diff-85c232ec82f12105613d6c349d87940c2b6dc21e6d77ad85964adc1c4a7b9183R38?

Reply:
I like the JSON a lot better than doing it in shell for sure